presto-main/src/main/java/com/facebook/presto/server/PrestoSystemRequirements.java

```diff
@@ -35,13 +36,15 @@
 {
     private static final int MIN_FILE_DESCRIPTORS = 4096;
     private static final int RECOMMENDED_FILE_DESCRIPTORS = 8192;
+    private static final List<String> JVM_VENDORS = ImmutableList.of(
+            "Azul Systems, Inc.", "Oracle Corporation");
 
     private PrestoSystemRequirements() {}
 
     public static void verifyJvmRequirements()
     {
         String vendor = StandardSystemProperty.JAVA_VENDOR.value();
-        if (!"Oracle Corporation".equals(vendor)) {
+        if (!JVM_VENDORS.contains(vendor)) {
```

Comment:
maybe instead of making it supported or failing, we could raise a warning that you are suing `Azul Systems, Inc.` which should be ok but it is not tested, so you are doing this on your own risk. 

I do not expect that we will start to run tests on two different JVMs in order we are ok with both of them. Either I do not expect that all the users would migrate to Azul ;)

Having such warning would be just good enough.

Reply:
Alternatively, we could have a config switch that skips this check.. But I guess there might be reasons that we don't have it yet, are there?

Reply:
@electrum I've updated the previously referenced [Travis CI config](https://github.com/dekobon/java-manta/blob/travis-ci-additional-jdks/.travis.yml) to use Java 9.

@kokosing A warning would solve the immediate problem, but it would be odd because the running JVM would be more compatible than most OpenJDK distributions. If that is the case, then wouldn't you also want to warn when running an OpenJDK?

@findepi A switch would also solve the problem.

More broadly, do you really need this check? I've never seen a Java project that does this check before. It is up to the user to choose a JVM and to vet it when running an open source Java application. Most projects publish in their documentation what JVMs they test with and that is sufficient.

Reply:
Sure. AFAIR we do not any tests for OpenJDK, so it sounds reasonable to have such warning for OpenJDK as well.

My understanding of this check is to prevent silly errors of cluster configuration which will cause misbehavior of Presto that would be hard to diagnose. That is why Presto output all the configuration when it starts and have such checks like that.

Reply:
Presto is a heavy user of the JVM and has hit many bugs with the JIT, so we enforce a minimum Java version that is known to work well. This version check is specific to OpenJDK. Does Azul use the same versioning, i.e., would 8u92 be the same on OpenJDK and Azul?